juju_verify/verifiers/__init__.py

```diff
@@ -50,27 +50,28 @@ def get_verifier(units: List[Unit]) -> BaseVerifier:
     charm_types = set()
 
     if not units:
-        raise CharmException('List of units can not be empty when creating '
-                             'verifier')
+        raise CharmException("List of units can not be empty when creating " "verifier")
```

Comment:
This does not seem right. It's cool that it found the unnecessarily split line, but the way it merged it is questionable :D

Reply:
Yes, this is really weird: D